src/Module.ts

```diff
@@ -39,11 +39,26 @@ import Literal from './ast/nodes/Literal';
 
 const setModuleDynamicImportsReturnBinding = wrapDynamicImportPlugin(acorn);
 
-export interface IdMap { [key: string]: string; }
+export interface IdMap {[key: string]: string;}
 
-interface CommentDescription { block: boolean, text: string, start: number, end: number }
-interface ExportDescription { localName: string, identifier?: string }
-interface ReexportDescription { localName: string, start: number, source: string, module: Module }
+interface CommentDescription {
+	block: boolean,
```

Comment:
I believe this should be a semi colon instead of a comma.

Not sure if that causes a build error or not. But at least that's the norm.